src/components/EditImageModal/index.jsx

```diff
@@ -120,8 +133,12 @@ export default class EditImageModal extends React.Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
+    if (this.state.isStatusAlertOpen) {
+      this.statusAlertRef.focus();
+    }
+
     if (this.state.pageNumber === 1 && prevState.pageNumber === 2) {
-      // TODO: add focus
+      this.dropZoneButtonRef.focus();
```

Comment:
I'm having some wet feet about this. If the original page was page 2 and the current page is page 1, that means either you got to page 2 and exited the modal or you got to page 2 and successfully uploaded an image, thereby closing the modal. I would kind of expect the focus to just go to the original location (x button on the modal) on opening the modal again. I guess I feel like I'd be caught kind of guard if the first time I open the modal, focus went to the x, and the second time, it went somewhere else. It doesn't seem as predictable, and I wonder whether it does a little too much assumption on our part about what the user wants. What do you think?